src/main/java/nextstep/optional/Expression.java

```diff
@@ -14,12 +16,9 @@ private static boolean matchExpression(Expression e, String expression) {
     }
 
     static Expression of(String expression) {
-        for (Expression v : values()) {
-            if (matchExpression(v, expression)) {
-                return v;
-            }
-        }
-
-        throw new IllegalArgumentException(String.format("%s는 사칙연산에 해당하지 않는 표현식입니다.", expression));
+        return Arrays.stream(Expression.values())
+                .filter(value -> matchExpression(value, expression))
+                .findFirst()
+                .orElseThrow(IllegalArgumentException::new);
```

Comment:
```suggestion
                .orElseThrow(() -> new IllegalArgumentException(String.format("%s는 사칙연산에 해당하지 않는 표현식입 
니다.", expression)));
```
알고 계시겠지만, 이렇게 사용할 수도 있습니다:)